lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/transport/netty/NettyMessagingTransport.java

```diff
@@ -95,8 +101,11 @@
   private final int serverPort;
   private final SocketAddress localAddress;
 
-  private final NettyClientEventListener clientEventListener;
-  private final NettyServerEventListener serverEventListener;
+  private final AbstractNettyEventListener clientEventListener;
+  private final AbstractNettyEventListener serverEventListener;
+
+  private final boolean ssl;
```

Comment:
use naming appropriate for a Boolean flag, e.g. `hasSsl` or `sslSupported` or something